src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -1766,14 +1768,77 @@ class AddJarSegment(BaseSegment):
     )
 
 
+@spark3_dialect.segment()
+class AnalyzeTableSegment(BaseSegment):
+    """An `ANALYZE {TABLE | TABLES}` statement.
+
+    https://spark.apache.org/docs/latest/sql-ref-syntax-aux-analyze-table.html
+    """
+
+    type = "analyze_table_statement"
+
+    match_grammar = Sequence(
+        "ANALYZE",
+        OneOf(
+            Sequence(
+                "TABLE",
+                Ref("TableReferenceSegment"),
+                Ref(
+                    "PartitionSpecGrammar",
+                    optional=True,
+                ),
+                "COMPUTE",
+                "STATISTICS",
+                OneOf(
+                    "NOSCAN",
+                    Sequence(
+                        "FOR",
+                        "COLUMNS",
+                        OptionallyBracketed(
+                            OneOf(
+                                Ref("ColumnReferenceSegment"),
+                                Delimited(
+                                    Ref(
+                                        "ColumnReferenceSegment",
+                                    ),
+                                ),
+                            ),
```

Comment:
Isn't this the same as:

```suggestion
                            Delimited(
                                Ref(
                                    "ColumnReferenceSegment",
                                ),
                            ),
```